bindings/go/evmc/evmc.go

```diff
@@ -149,6 +149,8 @@ const (
 	Istanbul         Revision = C.EVMC_ISTANBUL
 	Berlin           Revision = C.EVMC_BERLIN
 	London           Revision = C.EVMC_LONDON
+	Shanghai         Revision = C.EVMC_SHANGHAI
+	maxRevision      Revision = C.EVMC_MAX_REVISION
```

Comment:
Why not also add latest stable?

Reply:
`maxRevision` is private and cannot be used outside of the `evmc` package.

Reply:
How is it private? Because lowercase?

Reply:
Yes. But made it public now together with `LatestStableRevision`. I don't want to overthink it.